cinnabar/wrangle.py

```diff
@@ -22,22 +23,28 @@ def read_csv(filepath: pathlib.Path) -> dict:
                 expt_block = False
                 calc_block = True
             if expt_block and len(line.split(",")) == 3 and line[0] != "#":
-                expt = ExperimentalResultFromCSV(*line.split(","))
+                ligand, DG, dDG = line.split(",")
+                expt = ExperimentalResult(ligand, float(DG), float(dDG))
                 raw_results["Experimental"][expt.ligand] = expt
             if calc_block and len(line.split(",")) == 5 and line[0] != "#":
-                calc = RelativeResultFromCSV(*line.split(","))
+                ligA, ligB, calc_DDG, mbar_err, other_err = line.split(',')
+
+                calc = RelativeResult(ligA.strip(), ligB.strip(),
+                                      float(calc_DDG), float(mbar_err),
+                                      float(other_err))
                 raw_results["Calculated"].append(calc)
     return raw_results
 
 
-class RelativeResultFromCSV(object):
-    def __init__(self, ligandA, ligandB, calc_DDG, mbar_error, other_error):
-        self.ligandA = str(ligandA).strip()
-        self.ligandB = str(ligandB).strip()
+class RelativeResult:
```

Comment:
Rather than `RelativeResult` and `ExperimentalResult` (and eventually `AbsoluteResult`) maybe it makes more sense to just have `Relative` and `Absolute` where the experimental/computational-ness of an object is just another field?